src/style.css

```
@@ -0,0 +1,99 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-size: 100px;
+  margin: 0;
+}
+
+.conteiner {
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+}
+
+.block {
+  display: flex;
+  min-width: 300px;
+  height: 300px;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-grow: 1;
+}
+
+.block-1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (min-width:900px) and (max-width:1200px) {
+  .conteiner {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .block {
+    width: 33vw;
+  }
+
+  .block-4 {
+    order: 2;
+  }
+
+  .block-5 {
+    order: 1;
+  }
+
+  .block-6 {
+    order: 0;
+  }
+
+}
+
+@media (min-width:600px) and (max-width:899px) {
+  .conteiner {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .block {
+    max-width: 50vw;
+  }
+
+  .block-3 {
+    order: 1;
+  }
+
+  .block-4 {
+
+    order: 0;
+  }
```

```suggestion
  .block-4 {
    order: 0;
  }
```
redundant empty line